tests/unit/Scaffold/PathResolverTest.php

```
@@ -258,6 +258,59 @@ public function testResolveProviderRootReturnsLockPathWhenInsideVendor(): void
+    {
+        $projectRoot = $this->tempDir . '/proj';
+        $vendor = $projectRoot . '/vendor';
+
+        mkdir($vendor, 0777, recursive: true);
+
+        $result = PathResolver::resolveProviderRoot(
+            $vendor,
+            'pkg/name',
+            ['path' => 'vendor/pkg/name-missing'],
+            $projectRoot . '/',
+        );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\yii\scaffold\Scaffold\PathResolver' in method 'testResolveProviderRootTrimsTrailingSeparatorFromProjectRootBeforeJoiningRelativeLockPath'. (undefined)

(StaticAccess)